docs/sites.yml

```diff
@@ -8381,6 +8381,29 @@
     - User Experience
   built_by: Binaria
   built_by_url: "https://binaria.com/"
+- title: Century 21 Financial
+  url: https://century21financial.co.nz/
+  main_url: https://century21financial.co.nz/
+  description: >
```

Comment:
Need to remove > and move the next line to this one

Reply:
Done